src/components/ContactForm.js

```diff
@@ -1,19 +1,23 @@
 import { Form, FormGroup, Label, TextInput, Textarea } from '@trussworks/react-uswds'
 import '@trussworks/react-uswds/lib/index.css'
 
+import { useTranslation } from "react-i18next";
+import "../translations/i18n";
```

Comment:
I think you can just import your i18n.js file in your index.js, so you don't have to import it in each file where you need translations